edxval/api.py

```diff
@@ -900,9 +918,12 @@ def import_from_xml(xml, edx_video_id, course_id=None):
                 profile_name
             )
             continue
+        url = encoded_video_el.get('url')
+        if url.startswith(EXPORTED_VIDEO_PREFIX):
+            url = url.replace(EXPORTED_VIDEO_PREFIX, IMPORTED_VIDEO_PREFIX)
```

Comment:
The problem with this is that `olx://video/<video-id>/<video>` will get stored in the DB as <code>local-video://**video/**\<video-id>/\<video></code>.  This is extraneous information, and complicates the video xblock code which currently has to clean up that prefix:

https://github.com/edx/edx-platform/blob/e5270f790/common/lib/xmodule/xmodule/video_module/video_module.py#L1140